src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -551,7 +551,9 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
             })
 
             setTxId(txId)
-            setSerializedTxIndex(serializeTxIndex(poolAssetAccountId, txId, accountAssetAddress!))
+            setSerializedTxIndex(
+              serializeTxIndex(poolAssetAccountId, txId, fromAccountId(poolAssetAccountId).account),
```

Comment:
🙏🏽 very nice find, UTXO Txs are indeed indexed by xpub, not `from` address (which could be many!)